pycoast/cw_base.py

```diff
@@ -60,6 +60,31 @@ def get_resolution_from_area(area_def):
         return "f"
 
 
+def coord_to_pixels(x, y, coord_ref: str, area_def):
```

Comment:
May I suggest to change the name of `x`, `y` in the function signature? Since the meaning of their value have different meaning depend on the `coord_ref`, I think it is better to use some generic names instead of `x, y` or `lon, lat` here.

Reply:
I agree with the change in the docstring, but maybe `x` and `y` are generic enough while still referring to the dimension/axis being used.

Reply:
I think x,y are the generic axis names.
I've changed the docstring and code to use x,y everywhere appropriate now.